src/libs/constants/constants.ts

```diff
@@ -1 +1,6 @@
 export { FIRST_ARRAY_ELEMENT } from "./first-array-element.constant.js";
+export {
+	REVIEWS,
+	REVIEWS_TO_RENDER,
+	getRandomReviews,
+} from "./reviews-array.constant.js";
```

Comment:
this is not a generic constant that will be used in different modules in our app, so it should not be placed inside of here. Think about a better place for that

Reply:
Also do not stuff all the things in one file, for helpers we have helpers folder, for types types folder etc